service/src/test/java/tds/assessment/repositories/impl/ItemQueryRepositoryImplIntegrationTests.java

```diff
@@ -82,10 +82,7 @@ public void setUp() {
                 "('(SBAC_PT)SBAC-SEG1-MATH-8-Spring-2013-2015', 528, 528, 'PracTest::MG8::S1::SP14', 'ENU', '187-528', NULL, 0, 8233, NULL, 'Default'),\n" +
                 "('(SBAC_PT)SBAC-SEG1-MATH-8-Spring-2013-2015', 529, 529, 'PracTest::MG8::S1::SP14::Braille', 'ENU-Braille', '187-529', NULL, 0, 8233, NULL, 'Default'),\n" +
                 "('(SBAC_PT)SBAC-SEG2-MATH-8-Spring-2013-2015', 532, 532, 'PracTest::MG8::S2::SP14::Braille', 'ENU', '187-532', NULL, 0, 8233, NULL, 'Default'),\n" +
-                "('(SBAC_PT)SBAC-SEG2-MATH-8-Spring-2013-2015', 534, 534, 'PracTest::MG8::S2::SP14', 'ENU-Braille', '187-534', NULL, 0, 8233, NULL, 'Default'),\n" +
```

Comment:
The test data was set up wrong which is why we didn't see this in our testing.  Basically, we were giving adaptive tests forms and form items.  Removed those so the (SBAC_PT)IRP-Perf-ELA-11-Summer-2015-2016 is the adaptive looking assessment